### PR TITLE
Fix collector CRD for kube 1.22

### PR DESCRIPTION
**Pull request by rkukura**
Signed-off-by: Robert Kukura <rkukura@redhat.com>

### CODE REVIEW

**Comment by pavolloffay** [comment]
@rkukura please sign the CLA

File: config/crd/kustomization.yaml [hunk 1]
```diff
@@ -7,10 +7,6 @@ resources:
 # +kubebuilder:scaffold:crdkustomizeresource
 
 patchesStrategicMerge:
-# patches here are for enabling the conversion webhook for each CRD
-- patches/webhook_in_opentelemetrycollectors.yaml
```
Comment (pavolloffay) [inline config/crd/kustomization.yaml:11]: please remove only this line and keep the scaffolding metadata in place

**Comment by VineethReddy02** [comment]
Fixes: #473

**Review by VineethReddy02 — CHANGES_REQUESTED** [review]
@rkukura thanks for the PR.

Can you please sign the CLA & address the comment from Pavol?

**Review by pavolloffay — APPROVED** [review]

File: config/crd/kustomization.yaml (hunk 1 above)
Comment (rkukura) [reply config/crd/kustomization.yaml:11]: Done in update

**Comment by rkukura** [comment]
> @rkukura thanks for the PR.
> 
> Can you please sign the CLA & address the comment from Pavol?

Should be all set.

**Review by VineethReddy02 — APPROVED** [review]
LGTM!

**Pull request merged**